core/server/worker/src/main/java/alluxio/worker/block/BlockLockManager.java

```diff
@@ -116,9 +117,12 @@ public long lockBlock(long sessionId, long blockId, BlockLockType blockLockType)
    * @param unit the time unit of the {@code time} argument
    * @return lock id or INVALID_LOCK_ID if not able to lock within the given time
    */
-  public OptionalLong tryLockBlock(long sessionId, long blockId, BlockLockType blockLockType,
+  public Optional<BlockLock> tryAcquireBlockLock(long sessionId, long blockId,
```

Comment:
after looking at the reference, I think we can even remove this method since it is only used in removeBlock, and seems like time out is unnecessary since we have RPC call timeout.

Reply:
I feel like we have problem defining the timeout behavior in general...so client might see unexpected behavior ex. client receive time out when trying to remove block but later on the block still gets removed.

Reply:
Let's refine this logic in another pr